### ISSUE TITLE
GOG language selection.

### ISSUE THREAD

**Issue by Giger22** [lutris/lutris#5308]
Lutris cannot download a proper language package. I downloaded Gothic 2 Gold and selected my language but it still installed an English language unlike like Heroic.

**Comment by strycore** [comment]
Did you use the installer from this page https://lutris.net/games/gothic-ii-gold-edition/ ?

**Comment by strycore** [comment]
I needed to merge some game pages for our GOG integration so the previous doesn't make sense. Was it the installer with "GOG installer; user-selectable resolution; fixes the "blank videos" issue" ?

**Comment by Giger22** [comment]
Yeah, tried again, this fixed blank videos issue but I did not have even an option to select a language unlike in Heroic or GameHub.

**Comment by strycore** [comment]
This is because whoever wrote the installer added flags to  make the GOG installer silent. This is bad practice and we try fix installers doing that but still a bunch are thinking they're being smartasses by doing that.

I'm going to remove the flags making the installer silent, this should give you the language selection.

**Comment by Giger22** [comment]
Ok, this added language selection in installation menu again but it look like this is issue with GOG. It looks like you have to specify language before downloading in this **special** case. I downloaded the Witcher I and you could just download the game and later specify a language in a menu. Gothic is an old game and probably does not support this.
[GameHub](https://imgur.com/w6URRge)
[Heroic](https://i.imgur.com/LP7XvOS.png)

**Comment by timteichmann** [comment]
The problem indeed occurs because the installer files are different per language. While I cannot offer a nice solution, I can provide a workaround that still allows to use the GOG installer script with selectable resolution:

1. Download the two installer files for the correct language from GOG manually
2. Create the directory `~/.cache/lutris/installer/gothic-ii/gog/`
3. Place the two downloaded files in that directory and rename them to `setup_gothic_2_gold_2.7_win10_(55878)-1.bin` and `setup_gothic_2_gold_2.7_win10_(55878).exe` (i.e. remove the `_(german)` part for example)
4. Start the GOG installer script in Lutris as you would normally do
5. In the installer step where you can choose whether to download or use a local file, select the local file `~/.cache/lutris/installer/gothic-ii/gog/setup_gothic_2_gold_2.7_win10_(55878).exe`
6. Proceed with the installer, et voila

(Note that the `~/.cache` directory is the default path for temporary installer files, if you changed that in the Lutris settings you might have to adjust the above workflow for the alternate path)

**Comment by strycore** [comment]
this should be resolved by the switch to gogdl (in addition to getting proper download speeds)

**Comment by Giger22** [comment]
> this should be resolved by the switch to gogdl (in addition to getting proper download speeds)

Is is left to an user or is it a dependency and will be solved in the next update by a developer?

**Comment by strycore** [comment]
it will come in a future update
